### DISCUSSION TITLE
10 Steps to Build a Top-Tier Discord Server for Your Open Source Community. ✨ | Glasskube.dev

### DISCUSSION THREAD

**Discussion by giscus** [glasskube/glasskube#601] [Blog]
# 10 Steps to Build a Top-Tier Discord Server for Your Open Source Community. ✨ | Glasskube.dev

The definitive guide to set up a Discord server your community members will really enjoy

https://glasskube.dev/blog/discord-setup/

**Comment by Smithdash** [comment]
create a discord server with, bots, verify and role
